app/src/main/res/layout/activity_main.xml

```diff
@@ -86,23 +101,33 @@
                 android:layout_height="wrap_content" />
         </RadioGroup>
 
-    <!-- TODO: H: constraintEnd to end of parent, with a 16dp marginEnd
+    <!--  : constraintEnd to end of parent, with a 16dp marginEnd
                V: constrainTop to bottom of calc_button -->
-        <TextView
-            android:id="@+id/num_pizzas_text_view"
-            android:text="@string/total_pizzas"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:textSize="24sp"/>
+    <TextView
+        android:id="@+id/num_pizzas_text_view"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="16dp"
+        android:layout_marginEnd="16dp"
+        android:text="@string/total_pizzas"
+
+        android:textSize="24sp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/calc_button" />
 
     <!-- TODO: H: constraintEnd to end of num_pizzas_text_view, constrainStart to start of num_pizzas_text_view
                V: constraintTop to bottom of hungry_radio_group, with a 16dp marginTop
                set layout_width to "0dp" so the constraints can set its width -->
-        <Button
-            android:id="@+id/calc_button"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:onClick="calculateClick"
-            android:text="@string/calculate" />
+    <Button
+        android:id="@+id/calc_button"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="16dp"
+        android:onClick="calculateClick"
```

Comment:
Remove the onClick reference here and replace it with a setOnClickListener in the MainActivity